blivet/populator/populator.py

```diff
@@ -64,6 +64,9 @@ def parted_exn_handler(exn_type, exn_options, exn_msg):
     if exn_type == parted.EXCEPTION_TYPE_ERROR and \
        exn_options == parted.EXCEPTION_OPT_YES_NO:
         ret = parted.EXCEPTION_RESOLVE_YES
+    elif exn_type == parted.EXCEPTION_TYPE_WARNING and \
+            exn_options & parted.EXCEPTION_RESOLVE_FIX:
+        ret = parted.EXCEPTION_RESOLVE_FIX
```

Comment:
I think this should set some flag and the installer should inform the user.

Reply:
I don't think Anaconda has a way to inform user about this change. We are already logging the exception so that should be enough for debugging.

It might be useful to add some callback that applications like blivet-gui could use to actually ask user, but that is something we can do in the future.

Reply:
Works for me :+1: